salalib/tigerp.h

```diff
@@ -38,7 +38,7 @@ class TigerCategory : public prefvec<TigerChain>
    TigerCategory() {;}
 };
 
-class TigerMap : public pqmap<std::string,TigerCategory>
+class TigerMap : public std::map<std::string,TigerCategory>

```

Comment:
This is dirty - the standard discourages deriving from stl types (various bits aren't virtual, so this could go wrong). Better to have a map as member.